src/views/tutorial-library/constants.ts

```diff
@@ -0,0 +1,40 @@
+import { productSlugsToNames } from 'lib/products'
+
+/**
+ * The Algolia index we are searching against for tutorials
+ */
+export const INDEX_NAME = 'prod_LEARN'
```

Comment:
We can also use `staging_LEARN` I think? for the preview stuff if we use this as an env.